src/router/index.js

```diff
@@ -29,6 +29,7 @@ const routes = [
     { path: '/name/:value', name: 'search-name', component: Search, props: { query: 'name' } },
     { path: '/search/:value', name: 'search-general', component: Search, props: { query: 'general' } },
     { path: '/tag/:value', name: 'search-tag', component: Search, props: { query: 'tag' } },
```

Comment:
This line might be obsolete. Of you define a new route `/tags/:value`, we can drop `/tag/:value` as I understand it.

Reply:
that’s correct. I did not remove it initially because I wasn’t sure if we would want to leave this route in there. Considering the discussion in the PR, I’ll just remove it.